themes/app/src/scss/components/_navigation.scss

```diff
@@ -72,11 +72,11 @@ $verticalGap: 15px;
     }
 }
 
-.main-nav__donate {
+.main-nav__button {
```

Comment:
nice refactor here :) good case of where `donate` is too specific and it modifiable text hence the use of `button` is good here